test/evaluation/markForSubmissionEvaluationHandler.spec.ts

```diff
@@ -67,13 +67,20 @@ describe('markForSubmissionEvaluationHandler tests', () => {
 
   it(`Mark input candidates for presentation submission`, () => {
     const inputCandidates: unknown = getFile('./test/dif_pe_examples/vp/vp_general.json');
-    const presentation: Presentation = new Presentation(inputCandidates['@context'], inputCandidates['presentation_submission'], inputCandidates['type'], inputCandidates['verifiableCredential'], inputCandidates['holder'], inputCandidates['proof']);
+    const presentation: VerifiablePresentation = {
+      '@context': inputCandidates['@context'],
+      presentationSubmission: inputCandidates['presentation_submission'],
+      type: inputCandidates['type'],
+      verifiableCredential: inputCandidates['verifiableCredential'],
+      holder: inputCandidates['holder'],
+      proof: inputCandidates['proof']
+    };
```

Comment:
instantiation should be cleaner with 'new'

Reply:
Nope it is not. You are looking at this with a Java hat on